### PR TITLE
Bump Weave Scope to v1.2.1

### PR DESCRIPTION
**Pull request by errordeveloper**

### CODE REVIEW

File: repo/packages/W/weavescope-probe/2/marathon.json.mustache [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
+        "hostPath": "/var/run/docker.sock",
+        "mode": "RW"
+      }
+    ],
+    "docker": {
+      "image": "{{resource.assets.container.docker.48be3cfb15b2382f4055c867ccf477ed9a2b22ada18813c5f0d398753be7bf27}}",
```
Comment (errordeveloper) [inline repo/packages/W/weavescope-probe/2/marathon.json.mustache]: As far as I understand this it's not strictl

File: repo/packages/W/weavescope-probe/2/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "packagingVersion": "3.0",
+  "name": "weavescope-probe",
+  "version": "1.2.1",
```
Comment (errordeveloper) [inline repo/packages/W/weavescope-probe/2/package.json]: If we were use `latests` here along with `latests_release` tag for the image, would it be okay or it's strongly preferred to have explicit versions wherever possible?

Comment (ryadav88) [inline repo/packages/W/weavescope-probe/2/package.json]: its preferred to have specific versions whenever possible.

File: repo/packages/W/weavescope-probe/2/marathon.json.mustache (hunk 1 above)
Comment (ryadav88) [inline repo/packages/W/weavescope-probe/2/marathon.json.mustache]: should be `resource.assets.container.docker.image` and the value defined in resource.json

Comment (errordeveloper) [inline repo/packages/W/weavescope-probe/2/marathon.json.mustache]: Ok, so I must had been mislead by some example earlier on.

File: repo/packages/W/weavescope-probe/2/package.json (hunk 2 above)
Comment (errordeveloper) [inline repo/packages/W/weavescope-probe/2/package.json]: It's just that we were debating if we could avoid having to open a PR for each version bump...

File: repo/packages/W/weavescope-probe/2/marathon.json.mustache (hunk 1 above)
Comment (errordeveloper) [inline repo/packages/W/weavescope-probe/2/marathon.json.mustache]: I've updated this.

File: repo/packages/W/weavescope/2/package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "packagingVersion": "3.0",
+  "name": "weavescope",
+  "version": "1.2.1",
+  "tags": ["microservices", "visualisation", "monitoring"],
+  "maintainer": "help@weave.works",
+  "description": "Live visualisation, monitoring and management\n\nWeave Scope produces a live interactive map of your containerized services and applications with no coding and no instrumentation",
```
Comment (ryadav88) [inline repo/packages/W/weavescope/2/package.json]: Please add a link to example at the end of description
Something like: Installation documentation: https://github.com/dcos/examples/tree/master/1.8/weavescope

Example is basically an installation guide. Please create a PR against https://github.com/dcos/examples to add the example and please make sure to use the template from other examples. These examples finally show up in https://dcos.io/docs/1.8/usage/tutorials/

File: repo/packages/W/weavescope-probe/2/package.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  "tags": ["microservices", "visualisation", "monitoring"],
+  "maintainer": "help@weave.works",
+  "description": "Live visualisation, monitoring and management\n\nWeave Scope produces a live interactive map of your containerized services and applications with no coding and no instrumentation",
```
Comment (ryadav88) [inline repo/packages/W/weavescope-probe/2/package.json]: Please add a link to example at the end of description
Something like: Installation documentation: https://github.com/dcos/examples/tree/master/1.8/weave_probe

Example is basically an installation guide. Please create a PR against https://github.com/dcos/examples to add the example and please make sure to use the template from other examples. These examples finally show up in https://dcos.io/docs/1.8/usage/tutorials/

File: repo/packages/W/weavescope/2/package.json (hunk 3 above)
Comment (errordeveloper) [inline repo/packages/W/weavescope/2/package.json]: @ryadav88 is there a chance we could merge this to bring the package up-to-date for existing users and we shall take the action to follow-up with documentation PR in a few weeks, what do you think?

Comment (ryadav88) [inline repo/packages/W/weavescope/2/package.json]: @errordeveloper A documentation example is helpful for new and existing users to understand how they can install and run a package on DC/OS. This is also now part of package submission guidelines: https://github.com/mesosphere/universe#submit-your-package
It shouldn't take a lot of your time as you can use the template the existing examples. 
Ex: https://github.com/dcos/examples/tree/master/1.8/flink

Comment (ryadav88) [inline repo/packages/W/weavescope/2/package.json]: @errordeveloper any update here?

**Comment by errordeveloper** [comment]
I am afraid it will take a while with out current workload...

**Comment by ryadav88** [comment]
@errordeveloper any update here?

**Comment by ryadav88** [comment]
@errordeveloper any update here? If you're not planning to work on it now, please close the PR. We can reopen when you want to revisit.

**Comment by OhRobin** [comment]
Closing the PR because of no response. If you would like to reopen it, please let us know.

**Pull request closed without merging**